test-engineering/load-tests/kubernetes-config/locust-worker-controller.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: "apps/v1"
+kind: "Deployment"
+metadata:
+  name: locust-worker
+  labels:
+    name: locust-worker
+spec:
+  replicas: 5
```

Comment:
Fun aside since I've never noticed this... Wonder if this is where we want to start tweaking if we want to run 10 worker instances instead of our current value. Or the tutorial also had https://cloud.google.com/architecture/distributed-load-testing-using-gke#scaling_up_the_number_of_users_optional for some scaling replicas/pods. Not relevant to this PR, but a :the-more-you-know: moment.

Reply:
This is done!